src/Shared/ServerCertificateValidationEventSource.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if !NETFRAMEWORK
+
+using System.Diagnostics.Tracing;
+
+namespace OpenTelemetry.ResourceDetectors;
+
+[EventSource(Name = "OpenTelemetry-ResourceDetectors-Http")]
```

Comment:
OpenTelemetry-ResourceDetectors-Http  -- is there a component with this name?

Reply:
No, I have created that as a separate name as a placeholder.

Reply:
We should not use shared name for the eventsources. It'll cause issues like https://github.com/open-telemetry/opentelemetry-dotnet/issues/4516#issuecomment-1566623475